SU-HSDO/stanford_mrc
# MRCD8-240 fixed postcard styles, responsive, etc.

# READY FOR REVIEW

# Summary
- Updated Postcard Horizontal styles
- Tweaked Postcard (vertical) styles to mirror mocks

# Needed By (Date)
- End of sprint (3/1)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Verify that the horizontal and vertical postcards look good and are responsive

# Affected Projects or Products
- MRC
- stanford_mrc

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/MRCD8-240

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
